oak_runtime/src/graph.rs

```diff
@@ -79,7 +79,7 @@ impl Runtime {
                     &mut s,
                     r###"    {} [label="{}" URL="{}"]"###,
                     node_id.dot_id(),
-                    node_info.name,
+                    NodeInfo::construct_debug_id(&node_info.name, *node_id),
```

Comment:
Is it possible to replace this call (and the other calls in this file) with `node_info.get_debug_id(*node_id)`?

Reply:
Would be, will do! :)